scripts/tests/twister/test_quarantine.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Tests for quarantine.py classes' methods
+"""
+
+import mock
+import os
+import pytest
+import textwrap
+
+from twisterlib.quarantine import QuarantineException, \
```

Comment:
```suggestion
from twisterlib.quarantine import (
    QuarantineException,
    QuarantineElement,
    QuarantineData
)
```
just to avoid flake8 issue
`continuation line over-indented for hanging indentFlake8(E126)`
(but this is only comment, we do not check flake8, only my internal check)